src/app/(admin)/admin/_components/ServiceProvider/Organization/DetailsEditOrg.js

```diff
@@ -0,0 +1,27 @@
+import { NumberInput } from 'react-admin';
+import PropTypes from 'prop-types';
+import { FormFieldWrapper } from '../../FormFieldWrapper';
+import { FormatOfWorkSelect } from '../FormatOfWorkSelect';
+
+const fieldGroupClass = 'flex w-full flex-col md:flex-row md:gap-6 [&>*]:flex-grow';
+
+export function DetailsEditOrg({ validate }) {
+  return (
+    <FormFieldWrapper title={'Деталі'} className="mt-3">
+      <div className={fieldGroupClass}>
+        <NumberInput
+          name={'yearsOnMarket'}
+          source={'yearsOnMarket'}
+          label={'Років на ринку'}
+          validate={validate}
+          min="0"
```

Comment:
I think for `NumberInput` it should be a number, not a string

Reply:
Thanks for comment! Set `type="number"`.